apis/externalsecrets/v1beta1/secretstore_types.go

```diff
@@ -185,6 +185,10 @@ type SecretStoreProvider struct {
 	// Infisical configures this store to sync secrets using the Infisical provider
 	// +optional
 	Infisical *InfisicalProvider `json:"infisical,omitempty"`
+
+	// Beyondtrust configures this store to sync secrets using Password Safe provider.
+	// +optional
+	Beyondtrust *BeyondtrustProvider `json:"beyondtrust,omitempty"`
```

Comment:
Except here. Don't use camel case here, this one is fine. :D

Reply:
Done